app/assets/stylesheets/_users.scss

```diff
@@ -0,0 +1,74 @@
+
+.user-list {
+  margin-bottom: 30px;
+}
+
+.user-list-item {
+  padding: 15px 150px 15px 0;
+  border-top: 1px solid #bfc1c3;
+  position: relative;
+}
+
+.user-list-item h3 {
+  margin-bottom: 10px;
+}
+
+.user-list-item h3 span {
```

Comment:
this should be sassed, i.e.
```
.user-list-item {
    ....
    h3 {
        ....
         span {
              ...
        }
   }
}

Reply:
or just use this? 🙈 
https://github.com/alphagov/notifications-admin/blob/master/app/assets/stylesheets/views/users.scss